Tasks/NuGetInstaller/task.json

```diff
@@ -79,25 +79,30 @@
                 "Normal": "Normal",
                 "Detailed": "Detailed"
             }
+        },   
+        {
+            "name": "nuGetVersion",
+            "type": "radio",
+            "label": "NuGet Version",
+            "defaultValue": "3.3.0",
+            "helpMarkDown": "The version of NuGet to use, or external verson.",
+            "required": "true",
+            "groupName": "advanced",
+            "options": {
+                "3.3.0": "3.3.0",
+                "3.5.0.1829": "3.5.0 – build 1829 (rc2)",
```

Comment:
That is incredibly specific.  Why simply put 3.5.x in the value and then pull the one you want based on the code that is there.